### ISSUE TITLE
Pytorch NN is not working?

### ISSUE THREAD

**Issue by qzhu2017** [sayred1/tf.keras_GCN#2]
@yanxon
It seems that there is a strange behavior in pytorch NN.linear method.
I built a 3-layer linear neuron model from SKlearn, the results are good (pytorch->`SK-ANN.py`).
But when I tried to do the same thing for pytorch, I just failed to obtain the same results. (`Torch_net.py`)

I think Howard met a similar issue before. Can you help me to resolve this issue?

**Comment by sayred1** [comment]
Still currently looking into it! With the skip connection and gg readout, I get better results, but the model never goes below an MAE of one. I'm putting together a training process similar to the AGCN repo where they train/optimize over batches of molecules as opposed to training/optimizing every specified epoch to see how the loss is fluctuating.

**Comment by qzhu2017** [comment]
NN is fixed. The problem is due to incorrect assignment of output shape.
